doc/src/modules/polys/reference.rst

```diff
@@ -180,13 +181,15 @@ Appell sequences
 Manipulation of rational functions
 ==================================
 
+.. module:: sympy.polys.rationaltools
```

Comment:
What does this do? It doesn't seem to have any effect on the document https://output.circle-artifacts.com/output/job/bf475de9-077a-4800-8b8c-48e96f34d599/artifacts/0/doc/_build/html/modules/polys/reference.html#module-sympy.polys.rationaltools